crates/astria-composer/justfile

```diff
@@ -0,0 +1,12 @@
+default:
+  @just --list
+
+set dotenv-load
+set fallback
+
+default_env := 'local'
+copy-env type=default_env:
+  cp {{ type }}.env.example .env
+
+run:
+  cargo run
```

Comment:
should this also include copying the example env (if .env does not exist)?

Reply:
We could but I like the explicit need to copy the environment. 

It's a two liner vs a one liner for first run, but makes configuration an explicit step.